client/rpc-api/src/state/mod.rs

```diff
@@ -136,4 +136,10 @@ pub trait StateApi<Hash> {
 	fn unsubscribe_storage(
 		&self, metadata: Option<Self::Metadata>, id: SubscriptionId
 	) -> RpcResult<bool>;
+
+	/// Execute block and return all storage traces
```

Comment:
Actually we don't need any special flag because we set up a dispatch as part of this. WRT wording, I think you're right it could be improved and I need to add some more details - will push a commit to address it.